packages/medusa-plugin-sendgrid/src/services/sendgrid.js

```diff
@@ -138,7 +138,7 @@ class SendGridService extends NotificationService {
       case "order.refund_created":
         return this.orderRefundCreatedData(eventData, attachmentGenerator)
       default:
-        return {}
+        return { ...eventData }
```

Comment:
todo: could we return eventData directly? or maybe is there a reason for copying the object first here?

Reply:
agreed